perception/sandbox/polygrasp/README.md

```diff
@@ -0,0 +1,48 @@
+# Polygrasp
+
+## Installation
+
+In a new conda environment, [install Polymetis](https://facebookresearch.github.io/fairo/polymetis/installation.html#simple).
+
+Install the package itself:
+
+```bash
+pip install -e .
+```
+
+Then, follow the instructions in the forked repos to create the necessary isolated conda environments:
+
+1. [UnseenObjectClustering](https://github.com/1heart/UnseenObjectClustering)
+1. [graspnet-baseline](https://github.com/1heart/graspnet-baseline)
+
+## Usage
+
+Make necessary configuration modifications. For example:
+- [conf/run_grasp.yaml](./conf/run_grasp.yaml) contains the configuration, e.g. robot IP.
+- [conf/masks/](./conf/masks/) contains a folder to define workspace masks for each camera, for each bin.
+- We expect calibrated camera parameters out of [eyehandcal](../eyehandcal).
+
+
+Ensure Polymetis is running on the machine connected to the robot. Then, start the necessary pointcloud/grasping/gripper servers:
```

Comment:
how?

Reply:
to be clear - how to make sure it's running?

Reply:
Bring up the polymetis server on the NUC connected to the Robot -- probably doesn't apply if you're running on Hello